biogeochem/EDCanopyStructureMod.F90

```diff
@@ -454,6 +473,12 @@ subroutine DemoteFromLayer(currentSite,currentPatch,i_lyr)
             
             cc_loss = currentCohort%excl_weight
             
+            leaf_c          = currentCohort%prt%GetState(leaf_organ,all_carbon_species)
```

Comment:
just want to point out here, for the point in the future when we actually do include isotopes, that while it is correct to sum over all carbon isotopes as here, it isn't actually consistent with the way that CLM/ELM do the isotopes.  There, the "carbon 12" is actually really equivalent to an "all carbon", and the 13C and 14C are ratioed relative to that.  So we'll have to think through how to make the correspondence correct when we link the models.

Reply:
hmmm, that is an interesting point.  I had been wondering about that.  It seems kind of strange to sum up the isotopes when C13 and C14 are such small constituents..